site/content/docs/contributing/1.0-roadmap.md

```diff
@@ -52,14 +68,20 @@ Longer Term goals include:
 persistent storage
 - Improving "kind as a library"
```

Comment:
This contradicts the beta requirements now. remove?

Reply:
ack, done